packages/minter-contracts/bin-ts/type-aliases.ts

```diff
@@ -0,0 +1,8 @@
+export type EnglishAuctionTezCodeType = { _type: "EnglishAuctionTezCodeType"; code: object };
```

Comment:
I think it makes more sense to define this at the top of each file so the generated files don't have any dependencies.

Reply:
I'm amenable to this. I'll make this change.